### PR TITLE
Move Actionview Helpers guides to seperate guide [skip ci]

### PR DESCRIPTION
**Pull request by p8**
### Summary

The ActionView Overview guide is currently a Work In Progress (WIP)
because, it seems, not all ActionView helpers are listed. As it's also
one of the largest guides we could move the helpers to a seperate guide.
This would allow us to remove the WIP status from the ActionView
Overview guide and reduce its size.

### CODE REVIEW

**Comment by jonathanhefner** [comment]
This seems like a good change to me, but I think it should probably be broken up into two PRs: first extract the helpers section without changing any of its content (and perhaps link to it in the original guide?), then replace the form-related portions with a shorter blurb and link to the appropriate guide.

**Comment by p8** [comment]
Yes, I 've deleted a lot :)

**Comment by p8** [comment]
@jonathanhefner This PR has been reduced to moving the Helpers Overview to a separate guide.

File: guides/source/action_view_overview.md [hunk 1]
```diff
@@ -402,1179 +401,7 @@ This will add `app/views/direct` to the end of the lookup paths.
...
-Returns meta tags "csrf-param" and "csrf-token" with the name of the cross-site
-request forgery protection parameter and token, respectively.
-
-```html
-<%= csrf_meta_tags %>
-```
-
-NOTE: Regular forms generate hidden fields so they do not use these tags. More
-details can be found in the [Rails Security Guide](security.html#cross-site-request-forgery-csrf).
+This overview has been moved to a separate guide: [Action View Helpers Guide](action_view_helpers.html).
```
Comment (jonathanhefner) [inline guides/source/action_view_overview.md:404]: What do you think about changing this "moved" notice into more of a segue at the end of this guide?

Something to the effect of: "Rails provides many helper methods to use with Action View.  For an overview of these helpers, see the [Action View Helpers Guide](action_view_helpers.html)."

The section title should also probably be changed to reflect its (lack of) content, e.g. just "Action View Helpers".

Comment (p8) [reply guides/source/action_view_overview.md]: @jonathanhefner I've changed this section. I've included some examples and the section now links to both the Action View Helpers and Action View Form Helpers guides.

Comment (p8) [reply guides/source/action_view_overview.md]: image

Comment (jonathanhefner) [reply guides/source/action_view_overview.md]: That looks great!

**Comment by jonathanhefner** [comment]
Thank you @p8!

**Comment by p8** [comment]
Thanks for the review!

**Pull request merged**